build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+  ext.versions = [
+      'kotlin': '1.2.31',
+      'jmh': '1.19',
+      'jsr305': '3.0.2',
+      'animalSniffer': '1.10',
+      'junit': '4.12',
+      'assertj': '1.7.0'
+  ]
+
+  ext.deps = [
+      'kotlin': [
+          'stdLib': [
+              'common': "org.jetbrains.kotlin:kotlin-stdlib-common:${versions.kotlin}",
```

Comment:
You don't need to specify the version, it's inherited from the plugin: https://kotlinlang.org/docs/reference/using-gradle.html#configuring-dependencies
> Starting with Kotlin 1.1.2, the dependencies with group `org.jetbrains.kotlin` are by default resolved with the version taken from the applied plugin.